packages/relay/src/formatters.ts

```diff
@@ -181,6 +181,10 @@ const prepend0x = (input: string): string => {
   return input.startsWith(EMPTY_HEX) ? input : EMPTY_HEX + input;
 };
 
+const trimPrecedingZeros = (input: string) => {
```

Comment:
Add tests for this method.
You should consider multiple inputs valid and invalid

Reply:
Pushed in the UTs!